CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## [v11.21.1](https://github.com/auth0/lock/tree/v11.21.1) (2020-02-03)
+[Full Changelog](https://github.com/auth0/lock/compare/v11.21.0...v11.21.1)
+
+
+**Added**
```

Comment:
If we stick to semver and this is a patch release, shouldn't this be listed as a `fix` instead?

Reply:
Yes, good spot. Let me update it